dpgen/auto_test/EOS.py

```diff
@@ -138,11 +149,14 @@ def make_confs(self,
                     output_task = os.path.join(path_to_work, 'task.%06d' % task_num)
                     os.makedirs(output_task, exist_ok=True)
                     os.chdir(output_task)
```

Comment:
provide variables POSCAR = 'POSCAR' or 'STRU' and POSCAR_orig = 'POSCAR.orig' or 'STRU.orig'